test/e2e/common_test.go

```diff
@@ -937,3 +1009,32 @@ func writeYaml(content string, fileName string) error {
 
 	return nil
 }
+func GetPort() int {
+	a, err := net.ResolveTCPAddr("tcp", "localhost:0")
+	if err != nil {
+		Expect("").To(Equal("Unable to find port"))
+	}
+
+	l, err := net.ListenTCP("tcp", a)
+	if err != nil {
+		Expect("").To(Equal("Unable to find port"))
+	}
+	defer l.Close()
+	return l.Addr().(*net.TCPAddr).Port
+}
+
+func ncz(port int) bool {
+	timeout := 500 * time.Millisecond
+	for i := 0; i < 5; i++ {
+		ncCmd := []string{"-z", "localhost", fmt.Sprintf("%d", port)}
+		fmt.Printf("Running: nc %s\n", strings.Join(ncCmd, " "))
+		check := SystemExec("nc", ncCmd)
+		fmt.Printf("Got: %s\n", check.OutputToString())
```

Comment:
This is a NOP: it will always be an empty string. Instead of this, how about adding `fmt.Println("All nc retries failed")` before the `return false`?

Reply:
shit, that is leftover debug.